python/create_mask.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+import glob
+import sys
+import mahotas
+import numpy as np
+import matplotlib.pyplot as plt
+
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+else:
+    path=''
+
+f=glob.glob(path+"/meta/*ch_location*.txt")
+
+file=open(f[0],"r")
+
+siz=4096
+iarr=np.zeros((siz,siz),dtype=np.byte)
+slate,circ=np.array(iarr),np.array(iarr)
+x,y=[],[]
+fill=0
+i=0
+
+for line in file:
+	if line[0:3] != 'Dat':
+		if line [0:3] == 'Sol':
+			r=line[21:-1]
+		elif line[0:3] == '   ':
+			x,y=np.append(x,int(float(line[6:13]))),np.append(y,int(float(line[20:27])))
```

Comment:
cast into a float to then do so into a int?
Also, what you gain having all that into a single line? is not readable...

Reply:
Ooohh.. is it because the numbers have dots?

Reply:
Line broken into 2 lines

The values read in come from a .txt file with some strings. Hence python considers them all strings and from what I have seen this is the best way to convert a string with decimals to a int. Doing just int(line[x:y]) causes the code to break down